Site/SiteAnalyticsModule.php

```diff
@@ -362,6 +387,111 @@ public function getFacebookPixelCommandsInlineJavascript()
 	}
 
 	// }}}
+	// {{{ public function hasBingUET()
+
+	public function hasBingUET()
+	{
+		return (
+			$this->bing_uet_id != '' &&
+			!$this->analytics_opt_out
+		);
+	}
+
+	// }}}
+	// {{{ public function pushBingUETCommands()
+
+	public function pushBingUETCommands(array $commands)
+	{
+		foreach ($commands as $command) {
+			$this->bing_uet_commands[] = $command;
+		}
+	}
+
+	// }}}
+	// {{{ public function prependBingUETCommands()
+
+	public function prependBingUETCommands(array $commands)
+	{
+		foreach ($commands as $command) {
+			array_unshift($this->bing_uet_commands, $command);
```

Comment:
This will result in `$commands` being added in the reverse order they were passed in which probably isn't what is intended.